webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js

```diff
@@ -82,7 +82,8 @@ class NodeDefSwitch extends React.Component {
                            edit={edit}
                            parentNode={parentNode}
                            nodes={nodes}
-                           container={this.element}/>
+                           container={this.element}
+                           showKeys={false}/>
```

Comment:
is NodeDefErrorBadge showKeys param always false?

Reply:
showKeys is not passed in nodeDefEntityForm, so in that case it's using the default value (true)

Reply:
but should we always hide the keys in NodeDefEntityForm as well? what do you think?